ejb/src/main/java/org/wildfly/httpclient/ejb/HttpDiscoveryHandler.java

```diff
@@ -39,14 +36,22 @@
 import java.util.concurrent.ExecutorService;
 
 /**
+ * Http handler for discovery requests.
+ *
  * @author <a href="mailto:tadamski@redhat.com">Tomasz Adamski</a>
  */
 
 public class HttpDiscoveryHandler extends RemoteHTTPHandler {
 
     private final Set<EJBModuleIdentifier> availableModules = new HashSet<>();
+    private final HttpServiceConfig httpServiceConfig;
 
+    @Deprecated
```

Comment:
For the future: For all these places where there's a github code review comment about the deprecation, there should be a code comment added.

Reply:
@bstansberry +1. I think it won't hurt adding a comment before merging this

Reply:
@bstansberry because Richard and I have been discussing how this will be configured, I decided to postpone deprecation for a future version. Right now I'll keep both versions of the methods I incremented non-deprecated.